app/controllers/application_controller.rb

```diff
@@ -27,7 +27,7 @@ def remove_www
   end
 
   def after_sign_out_path_for(_resource_or_scope)
-    request.referrer
+    request.referer
```

Comment:
lol, is this really caught by Rubocop? That's pretty funny actually

Reply:
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Rails/RequestReferer